pi4j-core/src/main/java/com/pi4j/io/pwm/Pwm.java

```diff
@@ -204,19 +204,19 @@ default Pwm on(Number dutyCycle, int frequency) throws IOException{
     }
 
     /**
-     *  Get the duty-cycle value as a decimal value that represents the
-     *  percentage of the ON vs OFF time of the PWM signal for each
-     *  period.  The duty-cycle range is valid from 0 to 100 including
-     *  factional values. (Values above 50% mean the signal will remain
-     *  HIGH more time than LOW.)
+     * Get the duty-cycle value as a decimal value that represents the
```

Comment:
Description "decimal value" and return type Integer don't match... I'm OK with the Integer if that correctly represents the value, but should match the JavaDoc...